### PR TITLE
Add ldas-tools-cmake

### PR DESCRIPTION
**Pull request by duncanmmacleod**
This PR adds `ldas-tools-cmake`, a utility package for the `ldastools` suite used in GW astrophysics.

### CODE REVIEW

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, this is now ready for review. @emaros has already privately agreed to be named as a maintainer.

File: recipes/ldas-tools-cmake/bld.bat [hunk 1]
```diff
@@ -0,0 +1,26 @@
+setlocal EnableDelayedExpansion
```
Comment (marcelotrevisani) [inline recipes/ldas-tools-cmake/bld.bat]: Why this is necessary?

File: recipes/ldas-tools-cmake/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+test:
+  commands:
+    - exit $(test -f ${PREFIX}/share/ldas-tools/cmake/Modules/LDASToolsInstallDirs.cmake)  # [not win]
+    - exit $(test -f ${PREFIX}/share/ldas-tools/cmake/Modules/Autotools/ArchiveX/cx_ldas_pkg_check.cmake)  # [not win]
+    - if not exist %LIBRARY_PREFIX%\share\ldas-tools\cmake\Modules\LDASToolsInstallDirs.cmake exit 1  # [win]
+    - if not exist %LIBRARY_PREFIX%\share\ldas-tools\cmake\Modules\Autotools\ArchiveX\cx_ldas_pkg_check.cmake exit 1  # [win]
+
+about:
+  home: https://git.ligo.org/ldastools/LDAS_Tools
+  license: Other
```
Comment (marcelotrevisani) [inline recipes/ldas-tools-cmake/meta.yaml]: Please add the ``license_file`` here

Comment (duncanmmacleod) [reply recipes/ldas-tools-cmake/meta.yaml]: @marcelotrevisani, the distribution of this code doesn't not have a license file, the spec file from which I've based a bunch of the metadata lists `license: ?`.

Comment (duncanmmacleod) [reply recipes/ldas-tools-cmake/meta.yaml]: this package only exists to service a number of downstream packages in the same series, which are all GPL, so if a license is _required_, I will release this under GPL.

File: recipes/ldas-tools-cmake/bld.bat (hunk 1 above)
Comment (duncanmmacleod) [reply recipes/ldas-tools-cmake/bld.bat]: perhaps its not? I have removed that line, so lets see what happens...

File: recipes/ldas-tools-cmake/meta.yaml (hunk 2 above)
Comment (emaros) [reply recipes/ldas-tools-cmake/meta.yaml]: Please list it as GPL 2+ as that is what the license should have been.

Comment (duncanmmacleod) [reply recipes/ldas-tools-cmake/meta.yaml]: @emaros, done in 472e298.

**Comment by duncanmmacleod** [comment]
@conda-forge/staged-recipes, I hope I have addressed the previous review comments from @marcelotrevisani, can this be reviewed again?

**Comment by marcelotrevisani** [comment]
Looks good to me! I'm going to merge this!

Thanks to put this in ``conda-forge`` !  :)

**Pull request merged**